lib/Doctrine/Common/Annotations/DocLexer.php

```diff
@@ -109,6 +109,14 @@ protected function getNonCatchablePatterns()
         return ['\s+', '\*+', '(.)'];
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function getModifiers()
+    {
+        return parent::getModifiers() . 'u';
```

Comment:
Should this maybe be solved upfront in the abstract lexer?

Reply:
I'll do it in an abstract lexer

Reply:
create PR https://github.com/doctrine/lexer/pull/35